tests/integration/interactive_init_test.go

```diff
@@ -81,6 +81,64 @@ var _ = Describe("odo init interactive command tests", func() {
 				Expect(helper.ListFilesInDir(commonVar.Context)).To(ContainElements("devfile.yaml"))
 			})
 
+			It("should allow for personalizing configurations", func() {
+				command := []string{"odo", "init"}
+				output, err := helper.RunInteractive(command, nil, func(ctx helper.InteractiveContext) {
+
+					helper.ExpectString(ctx, "Select language")
+					helper.SendLine(ctx, "Javascript")
+
+					helper.ExpectString(ctx, "Select project type")
+					helper.SendLine(ctx, "")
+
+					helper.ExpectString(ctx, "Select version")
+					helper.SendLine(ctx, "")
+
+					helper.ExpectString(ctx, "? Select container for which you want to change configuration?")
+					helper.SendLine(ctx, "runtime")
+
+					helper.ExpectString(ctx, "? What configuration do you want change?")
+					helper.SendLine(ctx, "Add new port")
+
+					helper.ExpectString(ctx, "? Enter port number:")
+					helper.SendLine(ctx, "3000")
+
+					helper.ExpectString(ctx, "? What configuration do you want change?")
+					helper.SendLine(ctx, "Delete port \"3000\"")
+
+					helper.ExpectString(ctx, "? What configuration do you want change?")
+					helper.SendLine(ctx, "Add new environment variable")
+
+					helper.ExpectString(ctx, "? Enter new environment variable name: ")
+					helper.SendLine(ctx, "DEBUG_PROJECT_PORT")
+
+					helper.ExpectString(ctx, "? Enter value for \"DEBUG_PROJECT_PORT\" environment variable:")
+					helper.SendLine(ctx, "5858")
+
+					helper.ExpectString(ctx, "? What configuration do you want change?")
+					helper.SendLine(ctx, "Delete environment variable \"DEBUG_PORT\"")
+
+					helper.ExpectString(ctx, "? What configuration do you want change?")
+					helper.SendLine(ctx, "NOTHING - configuration is correct")
+
+					helper.ExpectString(ctx, "? Select container for which you want to change configuration? ")
+					helper.SendLine(ctx, "")
+
+					helper.ExpectString(ctx, "? Which starter project do you want to use?")
+					helper.SendLine(ctx, "nodejs-starter")
+
+					helper.ExpectString(ctx, "? Enter component name:")
+					helper.SendLine(ctx, "my-nodejs-app")
+
+					helper.ExpectString(ctx, "Your new component 'my-nodejs-app' is ready in the current directory.")
+
+				})
+				Expect(err).To(BeNil())
+				Expect(output).To(ContainSubstring("odo init --name my-nodejs-app --devfile nodejs --devfile-registry DefaultDevfileRegistry --devfile-version 2.1.1 --starter nodejs-starter"))
+				Expect(output).To(ContainSubstring("Your new component 'my-nodejs-app' is ready in the current directory."))
+				Expect(helper.ListFilesInDir(commonVar.Context)).To(ContainElements("devfile.yaml"))
```

Comment:
You need to assert the devfile now contains the new port and the new env var, and does not contain the old port and env var; you can use `helper.FileShouldContainSubstring`  and `helper.FileShouldNotContainSubstring` functions.